.github/workflows/cache_data.yaml

```
@@ -33,7 +33,8 @@ jobs:
           gmt which -Ga @earth_relief_10m_p @earth_relief_10m_g \
                         @earth_relief_30m_p @earth_relief_30m_g \
                         @earth_relief_01d_p @earth_relief_01d_g \
-                        @earth_relief_05m_p @earth_relief_05m_g
+                        @earth_relief_05m_p @earth_relief_05m_g \
+                        @N35E135.earth_relief_03s_g.nc
```

It's OK using the internal name, but at least we should add a comment to better explain it.
```suggestion
                        @earth_relief_05m_p @earth_relief_05m_g
          # Download one tile of the 03s srtm data.
          # @N35E135.earth_relief_03s_g.nc is for internal use only.
          # The naming scheme may change. 
          # DO NOT USE IT IN SCRIPTS.         
          gmt which -Ga @N35E135.earth_relief_03s_g.nc
```